### ISSUE TITLE
Using Android2Po for easy translations

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/OpenFoodFacts-androidApp#49]
https://github.com/miracle2k/android2po

- `easy_install android2po`
- `a2po COMMAND --android myproject/res --gettext myproject/locale`
- `a2po init de fr`

**Comment by teolemon** [comment]
I'll then set up a git import to Launchpad. That way, we can capitalize on the existing translations, and translator community.

**Comment by teolemon** [comment]
commited initial gettext export and imported the translations to Launchpad.
Translation work ongoing >> https://translations.launchpad.net/openfoodfacts/trunk/+pots/openfoodfacts-android-native

**Comment by itchix** [comment]
Added to the README

**Comment by nebulon42** [comment]
The link mentioned in the readme and here does not work (anymore). Where is the Android translation project?

**Comment by teolemon** [comment]
@nebulon42 https://translations.launchpad.net/openfoodfacts/openfoodfacts-android-native (for the App)
https://translations.launchpad.net/openfoodfacts/trunk/+pots/common-web (for the Server, also needed)

**Comment by nebulon42** [comment]
Thanks.

**Comment by nebulon42** [comment]
I had a go at the German translations for both Android app and server. I noticed inconsistencies regarding formal or informal language. I made sure that for the server all strings are in formal language. For the Android app now all strings are in informal language, but this should probably be changed to match the server. How is this handled in French or other languages?

Another thing: I noticed that the note regarding self-made photos is not present in the Android app. I think this is important. Should this be a separate bug report?
